quattor/functions/filesystem.pan

```diff
@@ -49,6 +49,32 @@ function lvs_add = {
     SELF;
 };
 
+@documentation{
+    desc = Adds a list of logical volumes to a volume group with mutiple lv options. To replace function lvs_add which only sets size.
+    arg = volume group, list of logical groups long with optons.  size and raid type are set if defined.
```

Comment:
I know this is following the convention of this file but I think this can be made clearer. There are probably more modern examples of how to document functions nicely elsewhere in the code base which maybe someone can point at but if not here's my suggestion:

```
    There are 2 mandatory arguments:
        1: string - the volume group name
        2: list of dicts representing the LVs desired. Supported keys within each dict are: size, raid_type
```

Reply:
thanks, I couldn't find an example of Modern way of doing this.  It appears  arg =  seems want all in one line, so added your suggestion by escaping it.